File: 07-Loops/secondQuestion.js
const input = require("readline-sync");

function repeat(amount){
    
    for(let index = 0; index < amount; index++){
        let userInput = input.questionInt("Enter a number: ");
        console.log(userInput);
    }
}

const loops = input.questionInt("Enter the amount of times you want to repeat: ");
repeat(loops);

File: 07-Loops/SecondQuestion.java
import java.util.Scanner;

public class SecondQuestion {
    public static void main(String[] args) throws Exception {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter how many loops: ");
        int loops = scanner.nextInt();
        repeat(loops);
    }

    public static void repeat(int amount){
        Scanner scanner = new Scanner(System.in);

        for (int index = 0; index < amount; index++) {
            System.out.println("Enter a number: ");
            int userInput = scanner.nextInt();
            System.out.println(userInput);
        }
    }
}
